lab_2_keywords_cooccurrence/start.py

```diff
@@ -27,6 +28,9 @@ def read_target_text(file_path: Path) -> str:
 
     # reading the text from which keywords are going to be extracted
     TARGET_TEXT_PATH_GENOME = ASSETS_PATH / 'genome_engineering.txt'
+    with open(TARGET_TEXT_PATH_GENOME, 'r', encoding='utf-8') as file:
```

Comment:
instead of opening the text yourself, use predefined `read_target_text` function